src/app/ongs/[slug]/templates/secondary/sections/more-info-about/index.tsx

```
@@ -0,0 +1,44 @@
+            key={`${section.title}-${index}`}
+          >
+            <article
+              className={`flex w-full flex-col gap-4 lg:gap-6 ${index % 2 === 0 ? 'lg:order-1' : 'lg:order-2'}`}
+            >
+              <h2 className="text-2xl font-bold lg:text-3xl">
+                {section.title}
+              </h2>
+              <div className="flex flex-col gap-4">
+                <p
+                  className="text-sm text-neutral-600 lg:text-base"
+                  dangerouslySetInnerHTML={{ __html: section.description }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)